static/src/javascripts-legacy/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -0,0 +1,166 @@
+define([
+    'bean',
+    'fastdom',
+    'Promise',
+    'lodash/functions/debounce',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/utils/storage',
+    'common/utils/mediator',
+    'common/utils/fastdom-promise',
+    'common/utils/private-browsing',
+    'text!common/views/experiments/tailor-survey.html',
+    'common/utils/fetch-json',
+    'lodash/collections/forEach'
+], function (
+    bean,
+    fastdom,
+    Promise,
+    debounce,
+    config,
+    cookies,
+    storage,
+    mediator,
+    fastdomPromise,
+    privateBrowsing,
+    quickSurvey,
+    fetchJson,
+    forEach
+) {
+    return function () {
+        this.id = 'TailorSurvey';
+        this.start = '2017-01-25';
+        this.expiry = '2017-02-31';
+        this.author = 'Manlio';
+        this.description = 'Testing Tailor surveys';
+        this.audience = 0.01;
+        this.audienceOffset = 0;
```

Comment:
Probably best to offset this so it interferes the least with other tests. you can check the current test on here: https://frontend.gutools.co.uk/analytics/abtests